spring-boot/admin/src/main/java/eu/coatrack/admin/security/PublicApiTokenAccessFilter.java

```diff
@@ -0,0 +1,27 @@
+package eu.coatrack.admin.security;
+
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationManager;
+import org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationProcessingFilter;
+import org.springframework.security.oauth2.provider.token.ResourceServerTokenServices;
+
+public class PublicApiTokenAccessFilter extends OAuth2AuthenticationProcessingFilter {
+
+    public PublicApiTokenAccessFilter(ResourceServerTokenServices resourceServerTokenServices) {
+
+        super();
+        setStateless(false);
```

Comment:
For debugging/testing, this setting is maybe helpful, but later we should probably set `stateless = true`, so that the public API only allows stateless calls with auth tokens.